src/chord/private/net/net.cpp

```diff
@@ -1,5 +1,4 @@
 #include "net/types.h"
```

Comment:
Unnecessary changes here also. I think there is no change for logging in this file.